en/integrations/plugin-packs/procedures/applications-ibmmq-mqi.md

```diff
@@ -0,0 +1,162 @@
+---
+id: applications-ibmmq-mqi
+title: IBM MQ MQI
+---
+
+## Pack Assets
+
+### Templates
+
+The Centreon Plugin Pack IBM MQ MQI brings 1 host template:
+* App-Ibmmq-Mqi-custom
+
+It brings the following Service Templates:
+
+| Service Alias | Service Template            | Default |
+|:--------------|:----------------------------|:--------|
+| Channels      | App-Ibmmq-Channels-Mqi      | X       |
+| Queue-Manager | App-Ibmmq-Queue-Manager-Mqi | X       |
+| Queues        | App-Ibmmq-Queues-Mqi        | X       |
+
+### Collected metrics & status
+
+<!--DOCUSAURUS_CODE_TABS-->
+
+<!--Channels-->
+
+| Metric Name                       | Unit   |
+|:----------------------------------|:-------|
+| status                            | string |
+| channel.traffic.in.bitspersecond  |        |
+| channel.traffic.out.bitspersecond |        |
+
+<!--Queue-Manager-->
+
+| Metric Name                    | Unit   |
+|:-------------------------------|:-------|
+| queuemanager.connections.count | count  |
+| status                         | string |
+
+<!--Queues-->
+
+| Metric Name                   | Unit  |
+|:------------------------------|:------|
+| queue.connections.input.count | count |
+| queue.message.oldest.seconds  |       |
```

Comment:
Maybe I'm wrong but it seems the metric does not return any unit. It should return "seconds" right ?https://github.com/centreon/centreon-plugins/blob/master/apps/mq/ibmmq/mqi/mode/queues.pm#L64

Reply:
We could set 's' in the Plugin but this kind of UOM is obsoleted by new naming convention. Don't bother adding it since force_new_perfdata is set to 1.